### DISCUSSION TITLE
Fix multiline issue with mysql slow query logs

### DISCUSSION THREAD

**Discussion by LinTechSo** [vectordotdev/vector#15960] [Q&A] [answered]
Hi all,
I want to ask if you could help or provide a working configuration for fixing multi line slow query logs in vector configurations

e.g
The slow query log is located at `/var/log/mysql/slow.log`

My problem is that i see slow query logs multi line in loki/elasticsearch sink.
these logs consumed from kafka source vector

i do not know how can i fix this multi line issue in vector transform .

i would appreciate if you could help me on this matter

thanks.

**Comment by spencergilbert** [comment, marked as answer]
Hi @LinTechSo 

It sounds like you'd want to use the `file` source and configure the [multiline](https://vector.dev/docs/reference/configuration/sources/file/#multiline) option. I'm not familiar with mysql's slow log formatting/style, but I'd start there.
